spyder_terminal/confpage.py

```diff
@@ -8,8 +8,10 @@
 
 # Standard library imports
 import os
+import platform
 
 # Third party imports
+from distutils.version import LooseVersion
```

Comment:
`distutils` is a standard library module, so please place it in the section above, not here.